tasks/task403_creak_commonsense_inference.json

```diff
@@ -6,7 +6,10 @@
         "CREAK 2018 task1 (\"https://arxiv.org/pdf/2109.01653.pdf\")"
     ],
     "Categories": [
-        "Classification"
+        "Classification -> Verification -> Statement Verification"
```

Comment:
Reasoning -> Commonsense Reasoning

Reply:
Are you suggesting to add commonsense reasoning as an extra category or replace the previous one?

Reply:
I suggest adding it as an extra category.